tests/test_fuzz.py

```diff
@@ -78,12 +78,24 @@ def _always_generate_random_port_WHEN_called():
 		compliant with RFC 6335, suitable for temporary testing purposes to
 		avoid port conflicts.
 
+		Should be equivalent to `return _random.randint(49152, 65535)`
+
 		Returns:
 			int: A random port number between 49152 and 65535.
 		"""
-		return _random.randint(49152, 65535)
+		return int(
+			49152 + (
+				int(
+					_random.SystemRandom().randbytes(
+						int(65535).__sizeof__()
+					).hex(),
+					16
+				) % 16383
+			)
+		)
 
 	@given(st.binary(min_size=1, max_size=1472))
+	@settings(deadline=None)
```

Comment:
ℹ️ This test can be impacted by underlying network and should not consider slow network a failing case. Disabling deadlines approved.